projects/step-frontend/src/lib/modules/_common/components/versions-dialog/versions-dialog.component.html

```diff
@@ -0,0 +1,42 @@
+<step-tabs [tabs]="tabs" [(activeTabId)]="activeTabId"> </step-tabs>
+
+<div class="tab-content">
+  <ng-container [ngSwitch]="activeTabId">
+    <ng-container *ngSwitchCase="VersionDialogTabs.FE">
+      <div class="list-item">
+        <strong class="flex-1">Package</strong>
+        <strong class="flex-1">Version</strong>
+      </div>
+
+      <div class="list-item">
+        <div class="flex-1">STEP</div>
+        <div class="flex-1">
+          <span style="display: none" id="frontend-os">{{ '${project.versionFEOS}' }}</span>
+          <span style="display: none" id="frontend-ee">{{ '${project.versionFEEE}' }}</span>
+        </div>
+      </div>
```

Comment:
```suggestion
      <div class="list-item" id="frontend-os">
        <div class="flex-1">Step base</div>
        <div class="flex-1">
          <span style="display: none">{{ '${project.versionFEOS}' }}</span>
        </div>
      </div>
    <div class="list-item"  id="frontend-ee">
        <div class="flex-1">Step enterprise</div>
        <div class="flex-1">
          <span style="display: none">{{ '${project.versionFEEE}' }}</span>
        </div>
      </div>
```

Reply:
Please check code if you accept the suggestion, I just wrote it untested. The idea would be that we have two Step versions in Step EE and it is super helpful for us to know which ones are deployed. Makes it more clear to have them in their own row

Reply:
And I think currently the version is not hidden when ${project.versionFEEE} is not replaced, correct?

Reply:
well I am not really sure, I used the snipped you've suggested and this is what I see 
![image](https://user-images.githubusercontent.com/8954749/201814771-afb49d5f-1487-4a1d-bfa8-6f485531b42c.png)

Reply:
however I am not really sure about those ids being attached to the list-item divs, are you sure this is how it should be? I can't really make an informed decision here

Reply:
![image](https://user-images.githubusercontent.com/8954749/201814960-c1f207fe-34dc-4743-a8cc-795723d1aa1c.png)

Reply:
You are right, we need to also pull up the display:none into the list-item level. And we also need the logic to hide them when the placeholder is not replaced or empty. I will check tomorrow with DevOps so we can hopefully test it.